src/lightning_app/cli/lightning_cli.py

```diff
@@ -57,7 +56,7 @@ def main():
 @click.group()
 @click.version_option(ver)
 def _main():
-    register_all_external_components()
```

Comment:
`register_all_external_components()`
Deleting this part also because it is not used anymore.